warehouse/utils/utils.go

```diff
@@ -612,6 +612,10 @@ func ObjectStorageType(destType string, config interface{}, useRudderStorage boo
 }
 
 func GetConfigValue(key string, warehouse model.Warehouse) (val string) {
+	configKey := fmt.Sprintf("Warehouse.%s.%s.%s", warehouse.Source.ID, warehouse.Destination.ID, key)
```

Comment:
[minor] prefix the config with a `pipeline` to distinguish for other types of warehouse configuration.

```suggestion
	configKey := fmt.Sprintf("Warehouse.pipiline.%s.%s.%s", warehouse.Source.ID, warehouse.Destination.ID, key)
```